rma/models/rma.py

```diff
@@ -878,6 +878,9 @@ def _create_receptions_from_picking(self):
         ).unlink()
         return_line = return_wizard.product_return_moves
         return_line.quantity = self.product_uom_qty
+        # The to_refund field is now True by default, which isn't right in the RMA
+        # creation context.
+        return_line.to_refund = False
```

Comment:
Do both in the same write for performance optimization.

Reply:
done